source/composition/structuredefinition-Composition.xml

```diff
@@ -180,6 +187,58 @@
         <map value="DocumentReference.version"/>
       </mapping>
     </element>
+    <element id="Composition.consent">
+      <path value="Composition.consent"/>
+      <short value="Refers to a FHIR Consent instance that indicates a patient's consent"/>
+      <definition value="Refers to a FHIR Consent instance that indicates a patient's consent."/>
```

Comment:
Should maybe ask in SD, but would suggest:

Short: Represents informed consents and medico-legal transactions
Definition: References the consents associated with this document. Examples include a consent to perform the clinical service being documented (.event) or a consent for the information contained in the document to be released to a third party

These come from https://hl7.org/cda/stds/online-navigation/infrastructure/cda/cda.htm#Consent